admin/ucf-degree-config.php

```diff
@@ -480,6 +494,19 @@ public static function settings_init() {
 					'options'     => self::get_api_select_options( self::get_profile_types() )
 				)
 			);
+
+			add_settings_field(
+				self::$option_prefix . 'update_tuition',
+				'Update Tuition Exceptions',
+				$display_fn,
+				$settings_slug,
+				$tuition_section,
+				array(
+					'label_for'   => self::$option_prefix . 'update_tuition',
+					'description' => 'When checked, the "Skip Tuition" setting will appear on degrees, allowing TuitionOverride objects to be created from the WP admin screen.',
```

Comment:
Something that clarifies what TuitionOverride objects are for those not familiar with the search service's data structure would be helpful in this description--e.g. 'When checked, the "Skip Tuition" setting will appear for degrees, which allows users to save per-degree tuition overrides to the UCF Search Service.'